components/script/dom/worker.rs

```diff
@@ -162,6 +162,7 @@ impl WorkerMethods for Worker {
         Ok(())
     }
 
+    // https://html.spec.whatwg.org/multipage/#handler-dedicatedworkerglobalscope-onmessage
     event_handler!(message, GetOnmessage, SetOnmessage);
     event_handler!(error, GetOnerror, SetOnerror);
```

Comment:
https://github.com/frewsxcv/servo/blob/macro-spec-links/python/tidy.py#L338

this checks two lines up for some reason...